### PR TITLE
Fix add doc for bias_attr

### PR DESCRIPTION
**Pull request by chengduoZH**

### CODE REVIEW

**Comment by luotao1** [comment]
LGTM https://github.com/PaddlePaddle/Paddle/pull/13937/commits/c18a6b529c63c33d38265bafd7a5402ed4c02d26

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -1278,8 +1278,12 @@ def sequence_conv(input,
         filter_size (int): the filter size (H and W).
         filter_stride (int): stride of the filter.
         padding (bool): if True, add paddings.
-        bias_attr (ParamAttr|None): attributes for bias
-        param_attr (ParamAttr|None): attributes for parameter
+        bias_attr (ParamAttr): The parameter attribute for the bias of this layer.
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py]: I prefer to add `If it is set to ParamAttr` to be complete

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -1545,7 +1551,8 @@ def conv2d(input,
     filter_shape = [num_filters, int(num_filter_channels)] + filter_size
 
     def _get_default_param_initializer():
-        std = (2.0 / (filter_size[0]**2 * num_channels))**0.5
+        filter_num_elem = filter_size[0] * filter_size[1] * num_channels
+        std = (2.0 / (filter_num_elem))**0.5
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py]: Is this a bug fix? seems quite bad. Can you put it in another PR and test the case? same for the one below.

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (chengduoZH) [reply python/paddle/fluid/layers/nn.py]: The doc has been refined.

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (chengduoZH) [reply python/paddle/fluid/layers/nn.py]: I will run this fix on CE.

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -1309,7 +1316,7 @@ def sequence_conv(input,
     return helper.append_activation(pre_act)
 
 
-def sequence_softmax(input, param_attr=None, bias_attr=None, use_cudnn=False):
+def sequence_softmax(input, use_cudnn=False):
```
Comment (qingqing01) [inline python/paddle/fluid/layers/nn.py]: Add `name` for in the interface.

**Review by qingqing01 — APPROVED** [review]
LGTM.

**Review by panyx0718 — APPROVED** [review]
